### PR TITLE
Update salinity_mission_execution.md

### PR DESCRIPTION
**Pull request by evibourma**
Some change in the structure of the document and text added. 
Regarding the paragraph "Gather data to help correct the sensor response time, i think it should be more or better explained, it is not exactly clear how will use the data.

### CODE REVIEW

File: sections/salinity_mission_execution.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 This section covers the activities of those deploying and recovering the gliders in the field in addition to best practices for glider pilots.
 
 (deploy)=
-## Deployment
+## Deployment and recovery
```
Comment (soerenthomsen) [inline sections/salinity_mission_execution.md:7]: As we try to keep the deployment life cycle as the main structure we should keep the old title her but link this section in the post-deployment section related to #29

**Comment by soerenthomsen** [comment]
Thanks a lot @evibourma for adding the picture! @isgiddy i suggest we first work up #29 to be able to link this section?

File: sections/salinity_mission_execution.md (hunk 1 above)
Comment (soerenthomsen) [reply sections/salinity_mission_execution.md:7]: You can see how we did this in the oxygen SOP in section 7.4 "Field calibration" 
https://oceangliderscommunity.github.io/Oxygen_SOP/sections/oxygen_post_recovery.html

Comment (soerenthomsen) [inline sections/salinity_mission_execution.md:7]:
```suggestion
## Deployment
```

**Review by isgiddy — APPROVED** [review]
Will merge now and edit to include the figure. Thanks @evibourma

**Pull request merged**